### PR TITLE
ts/indicators-chaikin

### LINKED ISSUE #11872: Migrate `indicator/chaikin` to TypeScript

### PR DESCRIPTION
**Pull request by activist**
Closes #11872, Tools: Migrated ChaikinIndicator to TS.

### CODE REVIEW

**Comment by activist** [comment]
Awaiting review and merge of #12183 before further work.

File: ts/indicators/chaikin.src.ts [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+                CHA: Array<[number, number]> = [],
+                xData: Array<number> = [],
+                yData: Array<number> = [],
+                periodsOffset: number,
+                // Shorter Period EMA
+                SPE: (boolean|Highcharts.IndicatorValuesObject),
+                // Longer Period EMA
+                LPE: (boolean|Highcharts.IndicatorValuesObject),
+                oscillator,
+                i;
```
Comment (bre1470) [inline ts/indicators/chaikin.src.ts]: `number` types are missing.

Comment (activist) [reply ts/indicators/chaikin.src.ts]: Ouch, missed those :( Thanks for noticing. Fixed now.

**Comment by activist** [comment]
> Conflicts in .eslintignore

Fixed.

**Pull request merged**